config/jobs/mkpj.sh

```diff
@@ -17,12 +17,10 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-TESTINFRA_ROOT=$(git rev-parse --show-toplevel)
+cd "$(git rev-parse --show-toplevel)"
 
-cd "${TESTINFRA_ROOT}"
-
-go run ./prow/cmd/mkpj/main.go \
-    --job-config-path=./config/jobs/ \
-    --config-path=./prow/config.yaml \
+bazel run //prow/cmd/mkpj -- \
```

Comment:
I wonder if it'd make sense to instead do this?

```
bazel run //config:mkpj -- --github-token-path=/dev/null "%@"
```

This keeps the specification of `--job-config-path` and `--config-path` in one place.

(I'm also not sure how useful specifying `--github-token-path=/dev/null` is. It silences a warning to stderr (that does not inhibit piping), but also presumably prevents actually passing it if you want it — but how often do you want it for k8s prow?)